meta/main.yml

```
@@ -18,22 +18,24 @@ galaxy_info:
   min_ansible_version: "2.10"
   namespace: cisagov
   platforms:
-    - name: Amazon Linux 2
-      versions:
-        - any
+    # Amazon Linux 2023 does not offer the following packages required by httpd:
+    # mod_auth_gssapi,mod_authnz_pam
```

We usually put a space after a comma- slightly more readable:
```suggestion
    # mod_auth_gssapi, mod_authnz_pam
```